Makefile

```diff
@@ -38,7 +38,11 @@ apply-with-kind: verify build-with-kind ## Deploy the kwok controller from the c
 		--set controller.image.tag=$(IMG_TAG) \
 		--set serviceMonitor.enabled=true \
 		--set-string controller.env[0].name=ENABLE_PROFILING \
-		--set-string controller.env[0].value=true
+		--set-string controller.env[0].value=true \
+		--set-string controller.env[1].name=KUBE_CLIENT_BURST \
+		--set-string controller.env[1].value=5000 \
+		--set-string controller.env[1].name=KUBE_CLIENT_QPS \
+        --set-string controller.env[1].value=5000
```

Comment:
nit: formatting

Also, should we just disable client-side throttling for the Kwok controller?